### PR TITLE
fix: workaround of LIMA_HOME usage

### LINKED ISSUE #10085: Add configuration for LIMA_HOME
### Is your enhancement related to a problem? Please describe

On macos env variables not inherited by GUI applications, so LIMA plugin cannot get `LIMA_HOME` env variable value to fetch info about lima vm

### Describe the solution you'd like

Add new configuration for plugin

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by tony-sol**
### Screenshot / video of UI

ext:lima
dash:lima
resources:lima

### What issues does this PR fix or reference?

Resolves https://github.com/podman-desktop/podman-desktop/issues/10085

### How to test this PR?

- [ ] Create lima virtual machine
- [ ] Place `LIMA_HOME` anywhere else than `HOME` dir
- [ ] Move `~/.lima` to new location defined in `LIMA_HOME`
- [ ] Open Settings->Preferences->Extension: Lima and set `LIMA_HOME` value to "Home" configuration
- [ ] Verify lima status on dashboard

### Additional info

Related to https://github.com/podman-desktop/podman-desktop/pull/10059#discussion_r1852562011

### CODE REVIEW

File: extensions/lima/package.json [hunk 1]
```diff
@@ -20,6 +20,11 @@
           "default": "",
           "description": "Custom path to limactl binary (Default is blank)"
         },
+        "lima.home": {
+          "type": "string",
+          "default": "",
```
Comment (feloy) [inline extensions/lima/package.json:25]: I think you will need to set the default as the value of LIMA_HOME if defined or `~/.lima` elsewhere

As an example, how it is done for Kubernetes' Kubeconfig file: https://github.com/podman-desktop/podman-desktop/blob/main/packages/main/src/plugin/kubernetes/kubernetes-client.ts#L224-L230

Comment (tony-sol) [reply extensions/lima/package.json:25]: Hmm, yeah, that's reasonable

Comment (benoitf) [reply extensions/lima/package.json:25]: it is not possible to define a default like that using package.json, so basically it's as in this PR. Blank = default value

**Review by benoitf — APPROVED** [review]

File: extensions/lima/package.json (hunk 1 above)
Comment (feloy) [reply extensions/lima/package.json:25]: ok, my bad. LGTM

**Review by feloy — APPROVED** [review]

**Comment by tony-sol** [comment]
imho, we should move this out from core into lima-extension [as well as docker-extension](https://github.com/podman-desktop/podman-desktop/pull/10059#discussion_r1852842415)

**Comment by benoitf** [comment]
@tony-sol AFAIK it has been applied to the lima extension (just that's a builtin extension)

**Pull request merged**